polar-core/tests/integration_tests.rs

```diff
@@ -1868,4 +1868,6 @@ fn test_builtin_iterables() {
         "x",
         vec![value!(1), value!(2)],
     );
+
+    qeval(&mut p, r#"forall(x in "abc", x in "abacus")"#);
```

Comment:
Does this only match b/c all of `"a"`, `"b"`, and `"c"` are in `"abacus"`? Could we add a complementary `qnull` test when that doesn't hold?

Reply:
Yep, will add a few more tests. Just wanted to push this before checkin so people could see the problem.

Reply:
@gj added